packages/strapi-helper-plugin/lib/src/components/Wysiwyg/helpers.js

```diff
@@ -0,0 +1,99 @@
+import { trimEnd, trimStart } from 'lodash';
+import styles from './styles.scss';
+/**
+ * Override the editor css
+ * @param  {[type]} block [description]
+ * @return {[type]}       [description]
+ */
+
+export function getBlockStyle(block) {
+  switch (block.getType()) {
+    case 'blockquote':
+      return styles.editorBlockquote;
+    case 'code-block':
+      return styles.editorCodeBlock;
+    default:
+      return null;
+  }
+}
+
+export function getBlockContent(style) {
+  switch (style) {
```

Comment:
Just asking why some values are uppercased (IMG, BOLD, ...) and some others are lowercased (blockquote, code-block)? We should keep the same format (lowercased) whatever the value is.

Reply:
Agree with you but the format should be uppercased

Reply:
Fixed in uppercase